### DISCUSSION TITLE
Auto updating from release tags

### DISCUSSION THREAD

**Discussion by SimonRichardson** [Homebrew/discussions#1728] [Tap maintenance and Homebrew development] [answered]
Previously there was an auto package releaser for published released tags.

We have [juju releases](https://github.com/juju/juju/releases/tag/juju-2.9.5) and the formula isn't being versioned correctly. Do we need something to magically do this again or has it moved back to a manual process?

**Comment by SMillerDev** [comment, marked as answer]
> Previously there was an auto package releaser for published released tags.

It still exists: https://github.com/dawidd6/action-homebrew-bump-formula

> We have juju releases and the formula isn't being versioned correctly. Do we need something to magically do this again or has it moved back to a manual process?

What do you mean by "versioned correctly"?

**Reply by SimonRichardson** [reply]
Just that the formula doesn't show the correct version.

**Reply by SMillerDev** [reply]
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/juju.rb shows the version as it was last updated. https://docs.brew.sh/Updating-Software-in-Homebrew documents how to submit updates to homebrew.
